bin/impl/setup-metrics.sh

```diff
@@ -94,14 +94,14 @@ $SED "/fluo.metrics.reporter.graphite/d" "$FLUO_PROPS"
   echo "fluo.metrics.reporter.graphite.frequency=30"
 } >> "$FLUO_PROPS"
 
-echo >&0 "Configuring InfluxDB..."
+print_to_console "Configuring InfluxDB..."
```

Comment:
These are fine, but one advantage of using the entire arg array rather than `$1` is that these quotes are no longer needed. It's not necessary to remove them, but it's nice to have the option to avoid quotes, in case we run into situations where we have to escape quotes in crazy ways.